ckanext/dcat/profiles.py

```diff
@@ -705,8 +705,11 @@ def parse_dataset(self, dataset_dict, dataset_ref):
             keywords.remove(keyword)
             keywords.extend([k.strip() for k in keyword.split(',')])
 
-        for keyword in keywords:
-            dataset_dict['tags'].append({'name': keyword})
+        # replace munge_tag to noop if there's no need to clean tags
+        do_clean = toolkit.asbool(config.get(DCAT_CLEAN_TAGS, False))
+        tags_val = [munge_tag(tag) if do_clean else tag[:100] for tag in keywords]
```

Comment:
I know this 100 limit is coming from the original CKAN harvester but in this case it would be a change in behaviour so let's keep `tag` as is if `'ckanext.dcat.clean_tags` is False (ie not trim it)